### ISSUE TITLE
Learning rate finder for Deep AR

### ISSUE THREAD

**Issue by mariosantosprivate** [awslabs/gluon-ts#1176]
Hello, I'd like to find the ideal learning rate for my Deep AR model and dataset, however after reading the available tutorials I'm not sure I understand how to.

In the "Usage" section of the Learning Rate Finder Tutorial it shows:

```python
net = mx.gluon.model_zoo.vision.resnet18_v2(classes=10)
learner = Learner(net=net, data_loader=data_loader, ctx=ctx)
lr_finder = LRFinder(learner)
lr_finder.find(lr_start=1e-6)
lr_finder.plot()
```
My question is regarding the `net` variable.
This is my model:
```python
deepAR_estimator = DeepAREstimator(
    prediction_length=prediction_length,
    context_length=2*prediction_length,
    freq="D",
    num_layers=10,
    num_cells=40,
    distr_output = NegativeBinomialOutput(),
    use_feat_dynamic_real=True,
    use_feat_static_cat=True,
    cardinality=stat_cat_cardinalities,
    dropout_rate=0.5,
    trainer=Trainer(   
        ctx = mx.context.gpu(),
        learning_rate=3e-4,
        epochs=400,
        num_batches_per_epoch=200,
        batch_size=64
    )
)
```
In this case is the correct usage to set `net = deepAR_estimator`, after having trained the model?

**Comment by parimuns** [comment]
Hello @mariosantosprivate 
I am facing a similar issue of finding a learning rate. Were you able to resolve this issue?

**Comment by mariosantosprivate** [comment]
Hello @parimuns sorry for the late reply!
I never managed to workaround this issue so I ended up going for another approach, seeing you can use learning_rate, learning_rate_decay_factor and minimum_learning_rate as parameters for your Trainer, you can end up with a pretty good learning rate as the model keeps training. Hope that helped!

**Comment by kekayan** [comment]
Hello @parimuns @mariosantosprivate ,
have you guys tried the `LearningRateReduction` callback?
```
callbacks = [
    LearningRateReduction(objective="min",
                          patience=10,
                          base_lr=1e-3,
                          decay_factor=0.5,
                          ),
]

estimator = DeepAREstimator(
    freq="H",
    prediction_length=6,
    context_length=12,
    dropout_rate=0.01,
    num_cells=80,
    num_layers=2,
    trainer=Trainer(#ctx = mx.context.gpu(),
                    epochs=50,
                    num_batches_per_epoch=10,
                    batch_size=25,
                    callbacks=callbacks))

```
